var/spack/repos/builtin/packages/tophat/package.py

```diff
@@ -29,6 +29,10 @@ class Tophat(AutotoolsPackage):
 
     parallel = False
 
+    def patch(self):
+        # Newer versions of autoconf hate calling AM_INIT_AUTOMAKE twice
```

Comment:
Given this comment, should the autoconf version be checked before performing the file filter operation?

Reply:
I _think_ calling it once is more correct regardless of version, but I'm not an autotools expert by far. I can hunt for the version where this changed though, if you'd like.